patient_data_upload/upload_vl_results.md

```diff
@@ -0,0 +1,31 @@
+# Uploading Viral Load Results
+## Downloading of Viral Load Results From VL Dashboard.
+1. Access the Viral load Dashboard via [vldash.cphluganda.org](https://vldash.cphluganda.org)
+![Viral Load Dashboard Page](../images/vl_dashboard_page.png)
+2. On the Top Menu Click on the The Login Item. This will navigate you to the login screen
+![Viral Load Dashboard Login Page](../images/vl_dashboard_login.png)
+3. Enter username and password and click the login page. This will navigate you to the Facility Dashboard you have logged into
+![Viral Load Facility Dashboard](../images/vl_facility_dashboard.png)
+4. On the Top Menu, Click on the _"Reports"_ menu Item. This will navigate you to the reports. 
+![Viral Load Reports Page](../images/vl_valid_results_page.png)
+5. Use The date filter to determine the viral load results in period you would like to download. filterout the viral load results for that period
+6. Click on the _"Valid Patients Results"_ Tab.
+7. Click on the Download Icon found at the bottom of the page.
+
+## Uploading Results in UgandaEMR.
+1. Login UgandaEMR. This will navigate you to the Home page.
+![UgandaEMR Home Page](../images/vl_ugandaemr_home page.png)
+2. On The Home page Click on the _"System Administrator"_ Icon. This will navigate you to System Administration Page. 
+![UgandaEMR System Administrator page](../images/vl_ugandaemr_system_admin.png)
+3. On the System Administrator page  click on the _"UgandaEMR Sync"_ Icon. This will navigate you to the UgandaEMR Sync Page. 
+![UgandaEMR Sync Page](../images/vl_ugandaemr_sync_page.png)
+4. On the UgandaEMR Sync Page Click on the _"Upload Viral Load Results"_ Icon. This will navigate you to the Upload Viral Load Results page. 
+![UgandaEMR Upload Results Page](../images/vl_upload_page.png)
+5. Click on the choose botton above
+
+6. Select CSV File to Upload
+
+7.  Click on the Upload Button
```

Comment:
@slubwama What feedback does a user get when the CSV file is uploaded? What about errors? What happens when the same file is uploaded twice?

Reply:
When you upload twice It will override the previous uploaded results in-other wards void the previous ones

Reply:
Done responding to this